streamdeck_ui/gui.py

```diff
@@ -1126,18 +1180,29 @@ def create_main_window(logo: QIcon, app: QApplication) -> MainWindow:
     return main_window
 
 
-def create_tray(logo: QIcon, app: QApplication, main_window: MainWindow) -> QSystemTrayIcon:
+def set_button_text_font_list(ui) -> None:
+    """Prepares the font selection combo box with all available fonts"""
+    ui.text_font.clear()
+    ui.text_font.setPlaceholderText("")
+    # the font list should only need to be set once
+    for i, font_family in enumerate(FONTS_DICT.keys()):
+        ui.text_font.addItem(font_family)
+        font = QFont(font_family)
+        ui.text_font.setItemData(i, font)
```

Comment:
ui.text_font.setItemData(i, font, Qt.FontRole)  # type: ignore [attr-defined]

Reply:
Config file blank, try to add new page broken
![#82-BrokenPageAdd](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/53057579/93fa67a3-150a-479c-922a-12688c1f3a63)

Reply:
it should be fixed now. please if you can confirm that it's fixed just mark as resolve the conversation. thank you in advance

Reply:
Yes All fixed now thanks

Reply:
Yup fixed thanks